main.go

```diff
@@ -2,8 +2,18 @@ package main
 
 import (
 	"fmt"
+	"html"
+	"log"
+	"net/http"
 )
 
 func main() {
-	fmt.Println("Main")
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "Hello %q", html.EscapeString(r.URL.Path))
+	})
+
+	err := http.ListenAndServe(":9000", nil)
```

Comment:
Can probably make it EVEN MORE CONDENSED:
`log.Fatal(http.ListenAndServe(":9000", nil))`

Since it's dummy small anyway, doesn't really matter haha. Just interesting:
https://golang.org/pkg/net/http/

Reply:
Yeah, I had seen both ways.
It'll get overwritten very soon anywa.